lib/compat/wordpress-6.5/fonts/README.md

```diff
@@ -0,0 +1,314 @@
+
+The only responsibility of a font collection is to present a list of font families in a correct format. Font collections are not in charge of installing the fonts; they just list a group of font families so users can choose the one they want to install. If the font family includes font faces, those font faces should have links to bundled (e.g: in a plugin assets) or external (e.g: from a CDN) font assets that will be downloaded or linked when the font family is installed.
+
+#### Register a font collection
+Register a font collection means providing a user a new list of fonts from one can select and install fonts. When a font collection is registered it is featured as a new tab in the font library modal and it's fetcheable using the `/font-collections/` REST API endpoint.
+
+To register a font collection you need to use `wp_register_font_collection` function.
+
+##### Register a font collection using PHP only
+Example:
+```php
+$font_families = [
+    array(
+        'font_family_settings' => (
+            array (
+                'fontFamily' => 'Piazzolla, serif',
+                'slug'       => 'paizzolla',
+                'name'       => 'Piazzolla',
+                'fontFace'   => array (
+                  array (
+                    'fontFamily' => 'Piazzolla',
+                    'fontWeight' => '400'
+                    'fontStyle'  => 'normal'
+                    'src'        => path_join( plugin_dir_path( __FILE__ ), "fonts/piazzolla400normal.woff2" )
+                  )
+                )
+            )
+        ),
+        'categories' => [ 'system-ui' ],
+    ),
+    array(
+        'font_family_settings' => (
+            array (
+                'fontFamily' => 'Arial, Helvetica, Tahoma, Geneva, sans-serif',
+                'slug'       => 'arial',
+                'name'       => 'Arial',
+            )
+        ),
+        'categories' => [ 'sans-serif' ],
+    ),
+];
+
+$categories = [
+    array(
+        'slug' => 'serif',
+        'name' => 'Serif',
+    ),
+    array(
+        'slug' => 'sans-serif',
+        'name' => 'Sans Serif',
+    ),
+];
+
+$config = array (
+    'name'          => 'My sytem fonts collection',
+    'description'   => 'Stacks of modern systems fonts, not font face assets needed. The look will vary on each system.',
+    'font_families' => $font_families,
+    'categories'    => $categories,
+);
+
+function register_my_collection(){
+    wp_register_font_collection ( 'my-font-foundry', $config );
+}
+
+add_action( 'init', 'register_my_collection' );
+```
+
+##### Register a font collection using a JSON file
+
+Example using a local JSON file:
+```php
+function register_my_collection(){
+    wp_register_font_collection ( 'my-font-foundry', path_join( __DIR__, 'my-collection.json' ) );
```

Comment:
```suggestion
    wp_register_font_collection (
        array(
            'name'          => 'my-font-foundry',
            'font_families' => path_join( __DIR__, 'my-collection.json' )
        )
    );
```
This method changed in https://github.com/WordPress/wordpress-develop/commit/8ced706f7f87a8882d5960b23b669d8e5217d54c.